guides/v2.1/cloud/live/stage-prod-migrate-prereq.md

```diff
@@ -41,7 +41,7 @@ You can locate your URLs through the Project Web Interface. For each selected en
 
 ## Pro plan projects {#pro}
 
-For Pro plan projects, make sure to complete all development and merging of your code to the `master` branch in the Integration environment. Only the `master` branch is deployed to Staging then Production.
+For Pro plan projects, you must merge your completed development code to the `integration` environment branch. Only the Integration environment branch can deploy to Staging, then Production.
```

Comment:
so there's `intergration` environment and  integration enviroment ...   Is our standard to just reference "integration environment" without the code styling?

Reply:
`integration` in miniscule code is an active branch name.
Integration with initial cap refers to the environment as a whole (PaaS).
The phrase "environment branch" may be a little confusing, but It is leftover from when each environment had a 'master' branch, of sorts.